content/docs/user-guide/how-to/run-dvc-on-windows.md

```diff
@@ -73,6 +73,39 @@ For VS Code, add
 
 to your global `settings.json` or to your project's `.vscode/settings.json`.
 
+### Set up LF Line Endings with .gitattributes for DVC on Windows
+
+To enhance DVC compatibility on Windows, it is advisable to employ a
+.gitattributes file with the eol attribute to configure line endings.
```

Comment:
.gitattributes -> `.gitattributes` (put into single backquotes to make it like an `inline code`)